assets/package.json

```diff
@@ -3,12 +3,14 @@
   "private": true,
   "engines": {
     "node": "~> 10.14",
-    "npm": "~> 6.9"
+    "npm": "~> 6.4"
   },
   "scripts": {
     "deploy": "npm run enforce-engine-versions && npx webpack --mode production",
     "watch": "npm run enforce-engine-versions && npx webpack --mode development --watch-stdin",
-    "enforce-engine-versions": "./scripts/enforce-engine-versions.js"
+    "enforce-engine-versions": "./scripts/enforce-engine-versions.js",
+    "preinstall": "npm run enforce-engine-versions",
+    "postinstall": "npm run enforce-engine-versions"
```

Comment:
Out of curiosity, why are we adding two hooks in the `install` lifecycle?

Reply:
Removing `preinstall` was a typo I made in #15 — it shouldn’t have been removed. I should have added `postinstall` as well (because `semver` might not be installed in `preinstall`).

https://github.com/mirego/elixir-boilerplate/pull/15/files#diff-84a003e0defbcf4bc4b690d3004222b8L11